code/__DEFINES/clothing.dm

```diff
@@ -98,7 +102,8 @@
 #define HAND_LEFT	512
 #define HAND_RIGHT	1024
 #define HANDS		1536
-#define FULL_BODY	2047
+#define NECK		2048
+#define FULL_BODY	4096 //2047

```

Comment:
Ah, so that's why it was that number, I wasn't sure why it was that, I'll change it as soon as I can.